server/src/main/java/org/candlepin/model/CertificateSerialCurator.java

```diff
@@ -88,7 +80,7 @@ public int deleteExpiredSerials() {
         List<String> ids = this.currentSession()
             .createCriteria(CertificateSerial.class)
             .add(Restrictions.le("expiration", Util.yesterday()))
-            .add(getRevokedCriteria())
+            .add(Restrictions.eq("revoked", true))
```

Comment:
This will delete those that have expired AND been marked as revoked. Is that the intention?